sections/Experiment.tex

```diff
@@ -33,13 +33,24 @@ \section{Experiment}\label{sec:experiment}
 This makes it hard to compare these two results on a single baseline.
 However, it is still possible to compare these two results between multiple baselines.
 
-We use \gls{map} to evaluate all baselines. 
-We also use precision@10 and precision@100 for the topic queries. \todo{describe what it is and how it is calculated}
-However since document queries only contain one ground truth positive, $p @ n$ doesn't make much sense to calculate. 
+We use \gls{map} to evaluate all baselines.
+\Gls{map} is calculated as follows\footnote{Equations from \url{https://towardsdatascience.com/breaking-down-mean-average-precision-map-ae462f623a52} and \url{https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)\#Mean_average_precision}}:
+
+$$MAP = \frac{1}{Q}\sum_{q=1}^{Q} AveP(q)$$
+
+\noindent Where $Q$ is the number of queries.
+
+$$AveP(q) = \frac{\sum_{n = 1}^{D} (precision@n(n) \times rel(n))}{|relevant~documents|}$$
+
+\noindent Where $D$ is the number of documents, and $rel(n)$ is 1 if the document at rank $n$ is relevant to the query, otherwise it is 0.
```

Comment:
```suggestion
\noindent Where $D$ is the number of documents, and $rel(n)$ is 1 if the document at or above rank $n$ is relevant to the query, otherwise it is 0.
```
Måske? Har ikke tjekket.